client/components/status/status.js

```diff
@@ -90,7 +90,9 @@ export default class Status extends React.Component {
       setLoading(false);
       if (isValid) {
         const justAuthenticated =
-          Object.keys(userData).length <= Object.keys(initialState).length;
+          sessionStorage.getItem(`${orgSlug}_just_authenticated`) === "true"
+            ? true
+            : Object.keys(userData).length <= Object.keys(initialState).length;
```

Comment:
It works.
I added the condition check for session storage because I want to know if a user is just authenticated or not i.e. if the user came after login then just authenticated will be true in the status page.